### DISCUSSION TITLE
챕터7.  CDN

### DISCUSSION THREAD

**Discussion by al-bur** [Frontend-Divers/performance-optimization#34] [함께 얘기하고 싶은 내용]
CDN

**Comment by al-bur** [comment]
서버와 클라이언트간의 거리가 웹 페이지 로딩 속도에 어떻게 영향을 미치는 지 궁금하네요

**Comment by al-bur** [comment]
P242 밑에 부분에 POP가 나오는데 이건 데이터 센터 같은 건지 궁금하네요

**Reply by ritajeong** [reply]
Point of Presence
둘 이상의 서로 다른 네트워크 또는 통신 장치가 서로 연결되는 지점
상호간 또는 개별에 대한 접속점 또는 접근점

**Comment by al-bur** [comment]
CDN에서 직접 웹 페이지 프론트엔드 최적화 기능을 제공해준다는데 우리는 그럼 성능 개선에 대한 공부를 할 필요가 없는 걸까요....ㅠㅠ

**Comment by Tanney-102** [comment]
롱테일 컨텐츠의 캐시 적중률이 캐시 계층을 더 둔다고 높아질까?
=> 애초에 필요한 일일까?

**Comment by Tanney-102** [comment]
CDN 동작 방법에서
DNS에 ip주소 요청 => 받은 응답으로 CDN사 DNS에 재요청 => ip 응답
인데
DNS에 대한 요청 수가 늘어나는게 거리가 먼 서버에 요청을 보내는 것 보다 오버헤드가 적은 것인지?

**Reply by al-bur** [reply]
요청 수 vs 거리

**Reply by Tanney-102** [reply]
실제 요청을 통해 수백Kb가 오가는 것에 비하면 dns 요청은 크게 고려하지 않아도 될 수준

**Comment by al-bur** [comment]
CDN 업체의 지표가 무엇일까

**Comment by al-bur** [comment]
국내에서만 제공되는 서비스에도 CDN이 필요할까?

**Reply by al-bur** [reply]
SSR이 필요할까?
